tests/SixLabors.Fonts.Tests/Issues/Issues_367.cs

```diff
@@ -23,7 +23,7 @@ public void ShouldMatchBrowserBreak()
         Assert.Equal(3, lineCount);
 
         FontRectangle advance = TextMeasurer.MeasureAdvance(text, options);
-        Assert.Equal(365, advance.Width);
+        Assert.Equal(355, advance.Width);
```

Comment:
The code you've added doesn't change the behavior. You've just moved the trimming function to a new method. Are there more changes to come?

Reply:
Yes, how embarrassing...I didn't push my last commit before the end of day

Reply:
Happens to the best of us 😄